dapr-http-client/src/Dapr/Client/HttpClient/Req.hs

```diff
@@ -21,14 +21,30 @@ makeHttpRequest ::
   Proxy response ->
   Option 'Http ->
   m (Either HttpException response)
-makeHttpRequest config method subUrl reqBody responseHandler options = runReq defaultHttpConfig $ do
+makeHttpRequestWithOptions config method subUrl reqBody responseHandler options = runReq defaultHttpConfig $ do
```

Comment:
I don't see the benefit by making a `makeHttpRequestWithOptions` to get rid of the `mempty`.
Then, Haskell code usually use `makeHttpRequest'` or `makeHttpRequest` for such kind of overload.

Reply:
Ok. I will remove this method then.